src/vm/instruction_set.md

```diff
@@ -481,6 +497,22 @@ Panic if:
 
 `$err` is cleared.
 
+### MLDV: Fused multiply-divide
+
+|             |                                                                                       |
+|-------------|---------------------------------------------------------------------------------------|
+| Description | Multiplies two registers with arbitrary precision, then divides by a third register.  |
```

Comment:
What does `arbitrary precision` refer to here? Does it mean the max value would be `u64::MAX * u64::MAX`?